_rules/SC1-2-1-video-only-alternative.md

```diff
@@ -0,0 +1,191 @@
+---
+name: video only has an accessibile alternative
+rule_type: composite
+description: |
+  This rule checks if video elements without audio have an alternative available
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+
+atomic_rules:
+- SC1-2-1-media-alternative-video
+- SC1-2-1-video-only-description-track
+- SC1-2-1-video-only-element-transcript
+- SC1-2-1-video-only-audio-alternative
+
+authors:
+- Wilco Fiers
+- Brian Bors
+- John Hicks
+- Rafal Charlampowicz
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video doesn't contain audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [media alternative video](SC1-2-1-media-alternative-video)
```

Comment:
These need to be full links/ url of the page.

Reply:
Fixed. Thanks.